python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -119,10 +121,9 @@ def start() -> None:
     # configure the environment
     os.environ["SERVICE_NAME"] = "performance_manager"
     validate_environment()
-    db_name = os.getenv("ALEMBIC_DB_NAME", "performance_manager")
 
     # run rds migrations
-    alembic_upgrade_to_head(db_name=db_name)
+    alembic_upgrade_to_head(db_name=os.getenv("ALEMBIC_DB_NAME"))
```

Comment:
I think `os.getenv` returns `None` if the env var is not found, and `alembic_upgrade_to_head` takes a string, so maybe have a dummy default to return if the `ALEMBIC_DB_NAME` env var is not found

```suggestion
    alembic_upgrade_to_head(db_name=os.getenv("ALEMBIC_DB_NAME", "db_not_found"))
```

Reply:
this should be ok because we will throw for a missing env var before we get here.